src/NLog.Web/NLogRequestPostedBodyMiddlewareConfiguration.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Text;
+using System.Web;
+
+namespace NLog.Web
+{
+    /// <summary>
+    /// Contains the configuration for the NLogRequestPostedBodyMiddleware
+    /// </summary>
+    public class NLogRequestPostedBodyMiddlewareConfiguration
+    {
+        /// <summary>
+        /// The default configuration
+        /// </summary>
+        public static readonly NLogRequestPostedBodyMiddlewareConfiguration Default = new NLogRequestPostedBodyMiddlewareConfiguration();
+
+        /// <summary>
+        /// Defaults to UTF-8
+        /// </summary>
+        public Encoding Encoding { get; set; } = Encoding.UTF8;
+
+        /// <summary>
+        /// Defaults to 1024
+        /// </summary>
+        public int BufferSize { get; set; } = 1024;
```

Comment:
Not sure this should be a setting, until someone requests this. I prefer to keep the visibility of methods and properties to absolute minimum, until someone needs access.

Reply:
Removed set to 1024 in the NLogRequestPostedBodyMiddleware class